docs/index.rst

```diff
@@ -79,54 +83,50 @@ Overview
-- A simple semantic layer, allowing users to control how data sources are
-  displayed in the UI by defining which fields should show up in which
-  drop-down and which aggregation and function metrics are made available
-  to the user
-- Integration with most SQL-speaking RDBMS through SQLAlchemy
-- Deep integration with Druid.io
+- An intuitive interface to explore and visualize datasets, and create interactive dashboards.
+- A wide array of beautiful visualizations to showcase your data.
+- Easy, code-free, user flows to drill down and slice and dice the data underlying exposed dashboards. The dashboards and charts act as a starting point for deeper analysis.
+- A state of the art SQL editor/IDE exposing a rich metadata browser, and an easy workflow to create visualizations out of any result set.
+- An extensible, high granularity security model allowing intricate rules on who can access which product features and datasets. Integration with major authentication backends (database, OpenID, LDAP, OAuth, REMOTE_USER, ...)
+- A lightweight semantic layer, allowing to control how data sources are exposed to the user by defining dimensions and metrics
+- Out of the box support for most SQL-speaking databases
+- Deep integration with Druid allows for Superset to stay blazing fast while slicing and dicing large, realtime datasets
+- Fast loading dashboards with configurable caching
+
 
 Databases
 ---------
 
 The following RDBMS are currently supported:
 
-- `Amazon Athena <https://aws.amazon.com/athena/>`_
-- `Amazon Redshift <https://aws.amazon.com/redshift/>`_
-- `Apache Drill <https://drill.apache.org/>`_
-- `Apache Druid <http://druid.io/>`_
-- `Apache Hive <https://hive.apache.org/>`_
-- `Apache Impala <https://impala.apache.org/>`_
-- `Apache Kylin <http://kylin.apache.org/>`_
-- `Apache Pinot <https://pinot.incubator.apache.org/>`_
-- `Apache Spark SQL <https://spark.apache.org/sql/>`_
-- `BigQuery <https://cloud.google.com/bigquery/>`_
-- `ClickHouse <https://clickhouse.tech/>`_
-- `CockroachDB <https://www.cockroachlabs.com/>`_
-- `Dremio <https://dremio.com/>`_
-- `Elasticsearch <https://www.elastic.co/elasticsearch/>`_
-- `Exasol <https://www.exasol.com/>`_
-- `Google Sheets <https://www.google.com/sheets/about/>`_
-- `Greenplum <https://greenplum.org/>`_
-- `IBM Db2 <https://www.ibm.com/analytics/db2/>`_
-- `MySQL <https://www.mysql.com/>`_
-- `Oracle <https://www.oracle.com/database/>`_
-- `PostgreSQL <https://www.postgresql.org/>`_
-- `Presto <http://prestodb.github.io/>`_
-- `Snowflake <https://www.snowflake.com/>`_
-- `SQLite <https://www.sqlite.org/>`_
-- `SQL Server <https://www.microsoft.com/en-us/sql-server/>`_
-- `Teradata <https://www.teradata.com/>`_
-- `Vertica <https://www.vertica.com/>`_
-- `Hana <https://www.sap.com/products/hana.html>`_
+- `Amazon Athena </installation.html#aws-athena>`_
+- `Amazon Redshift </installation.html#database-dependencies>`_
```

Comment:
What's the value in linking to the generic page here instead of the more specific db website?

Reply:
The "database dependencies" table has some information re installation for all the db options, but some db options have their own dedicated anchor tag (I'm not sure why the docs are written this way...). For the ones that have their own tag, it's link to there, but ones that don't, the link points to the "database dependencies" table for community users, which is more helpful than the homepage of the db itself with useful info as it relates to Superset.